README.md

```diff
@@ -9,21 +9,38 @@ Fleet Telemetry is a server reference implementation for Tesla's telemetry proto
 
 The service handles device connectivity as well as receiving and storing transmitted data. Once configured, devices establish a WebSocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
 
-## Configuring and running the service
+## Requirements
+These are the minimum system requirements for a Fleet Telemetry server, with 10 active vehicles connected, running on a Debian-based server with regular day-to-day usage. The requirements will change based on vehicle usage and quantity of vehicles.
+
+* Allocate at least 0.2 AWS vCPU, 256MB RAM, and 8GB storage
+* Host on a publicly available server with an associated domain
+
+### Recommendations
+* Run on a Debian base image
+* Allocate and configure a server certificate using best practices
+
+### Dependencies
+* Go 1.20+
+* libzmq
+* Docker*
+* Kubernetes/Helm*
+
+*Kubernetes/Helm is required when using the Helm Charts deployment method (recommended) and Docker is required for deploying Fleet Telemetry via a docker image.
 
-As a service provider, you will need to register a publicly available endpoint to receive device connections. Tesla devices will rely on a mutual TLS (mTLS) WebSocket to create a connection with the backend. The application has been designed to operate on top of Kubernetes, but you can run it as a standalone binary if you prefer.
+### Vehicle Compatibility
 
-### Install on Kubernetes with Helm Chart (recommended)
-For ease of installation and operation, we recommend running Fleet Telemetry on Kubernetes. Helm Charts help you define, install, and upgrade applications on Kubernetes. You can find a reference helm chart [here](https://github.com/teslamotors/helm-charts/blob/main/charts/fleet-telemetry/README.md).
+Vehicles must be running firmware version 2023.20.6 or later. You may find this information under your VIN in the Tesla Mobile App or the Software tab in the vehicle's infotainment system. Some older model S/X are not supported.
 
-### Install manually (skip this if you have installed with Helm on Kubernetes)
-1. Allocate and assign a [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name). This will be used in the server and client (vehicle) configuration.
+## Configuring and running the service
+You may generate [self-signed certificates](https://en.wikipedia.org/wiki/Self-signed_certificate) for your domain and the generated tls certificate + private key pair will be used to authenticate vehicles to your Fleet Telemetry server. Tesla vehicles rely on a mutual TLS (mTLS) WebSocket to create a connection with the backend.
```

Comment:
You can use a self signed certificate to start the server, but `/api/1/vehicles/fleet_telemetry_config` will reject anything not signed by Tesla's CA. I do not believe this line is accurate in my experience.

Reply:
Can you post what error you are receiving? I used a self-signed certificate for the fleet-telemetry server and it should be working as long as it registers properly with the endpoint.

Reply:
> Can you post what error you are receiving? I used a self-signed certificate for the fleet-telemetry server and it should be working as long as it registers properly with the endpoint.

`tesla_fleet_api.exceptions.Forbidden: {'response': None, 'error': 'Please follow instructions https://developer.tesla.com/docs/fleet-api#fleet-telemetry to send CSR to Tesla', 'error_description': ''}`

Reply:
Hi Brett
The error is because as discussed https://github.com/teslamotors/fleet-telemetry/issues/41#issuecomment-1875856991. 
We are onboarding new csr submissions incrementally.

Reply:
> Hi Brett
> The error is because as discussed https://github.com/teslamotors/fleet-telemetry/issues/41#issuecomment-1875856991. 
> We are onboarding new csr submissions incrementally. 

That's correct, so I am trying to point out that documentation that says you can use a self signed certificate is incorrect.

Reply:
Hmm I don't think thats the issue. If you have mailed your CSR and haven't heard back, that means your account isn't ready for step 3 as mentioned in https://developer.tesla.com/docs/fleet-api#fleet-telemetry